server/game/cards/plots/02/politicaldisaster.js

```diff
@@ -26,17 +31,17 @@ class PoliticalDisaster extends PlotCard {
 
     doDiscard() {
         _.each(this.selections, selection => {
-            var player = selection.player;
-            var toDiscard = _.difference(player.filterCardsInPlay(card => card.getType() === 'location'), selection.cards);
+            let player = selection.player;
+            let toDiscard = _.difference(player.filterCardsInPlay(card => card.getType() === 'location'), selection.cards);
 
             _.each(toDiscard, card => {
                 player.discardCard(card);
             });
 
             if(_.isEmpty(toDiscard)) {
-                this.game.addMessage('{0} does not discard any locations with {1}', player, this);
+                this.game.addMessage('{0} does not have any locations discarded for {1}', player, this);
```

Comment:
Sure. Personally I'd like to move away from the "[user] uses [card] to do [things]" template for effects where the [user] did not have any choice in the matter.